spec/dealership_spec.rb

```diff
@@ -0,0 +1,62 @@
+require './lib/car'
+require './lib/dealership'
+
+
+RSpec.describe Car do
+  context 'Iteration II' do
+    let (:dealership) {Dealership.new("Acme Auto", "123 Main Street")}
+    let (:car_1) {Car.new("Ford Mustang", 1500, 36)}
+    let (:car_2) {Car.new("Toyota Prius", 1000, 48)}
```

Comment:
Like the use of `let` here to set up all of your tests so that you don't need to repeat yourself